packages/core-blockchain/lib/index.js

```diff
@@ -11,12 +12,15 @@ exports.plugin = {
   defaults: require('./defaults'),
   alias: 'blockchain',
   async register (container, options) {
-    const blockchain = new Blockchain(options.networkStart)
+    blockchain = new Blockchain(options.networkStart)
 
     if (!process.env.ARK_SKIP_BLOCKCHAIN) {
       await blockchain.start()
     }
 
     return blockchain
+  },
+  async deregister (container, options) {
+    await blockchain.stop()
```

Comment:
No need for that wonky variable solution, can simply do `await container.resolve('blockchain').stop()`.

Reply:
Thanks @faustbrian, I've updated it with your suggestion